translate/cloud-client/hybrid_glossaries/hybrid_tutorial.py

```diff
@@ -0,0 +1,227 @@
+# Copyright 2019 Google LLC. All Rights Reserved.
```

Comment:
Sorry, the line should just be `# Copyright 2019 Google LLC`, the `. All Rights Reserved.` are from an older LICENSE format.